Dockerfile.agent

```diff
@@ -1,35 +1,47 @@
 # This file is for internal experimental purposes only.
```

Comment:
This file should not be updated, in reality it should even be deleted: it's relying on `9.0.0` artifacts and cannot be used for 8.* branches